.github/workflows/ci-tests.yml

```diff
@@ -201,7 +201,7 @@ jobs:
         pip install tox
     - name: Run the tests
       run: |
-        tox -e ${{ matrix.dbt-version }} -- plugins/sqlfluff-templater-dbt
+        tox -e ${{ matrix.dbt-version }} -- --cov=sqlfluff_templater_dbt plugins/sqlfluff-templater-dbt
```

Comment:
As mentioned above, this change means now we're checking coverage for the dbt templater, too.